dough-protection/src/main/java/io/github/bakedlibs/dough/protection/modules/LandsProtectionModule.java

```diff
@@ -33,19 +34,19 @@ public Plugin getPlugin() {
 
     @Override
     public void load() {
-        this.landsIntegration = new LandsIntegration(plugin);
+        this.landsIntegration = LandsIntegration.of(plugin);
     }
 
     @Override
     public boolean hasPermission(OfflinePlayer p, Location l, Interaction action) {
-        LandWorld landWorld = landsIntegration.getLandWorld(l.getWorld());
+        LandWorld landWorld = landsIntegration.getWorld(l.getWorld());
 
         if (landWorld == null) {
             return true;
         }
 
         Area area = landWorld.getArea(l);
-        return area == null || area.hasFlag(p.getUniqueId(), convert(action));
+        return area == null || area.hasRoleFLag(p.getUniqueId(), convert(action));
```

Comment:
typo?
```suggestion
        return area == null || area.hasRoleFlag(p.getUniqueId(), convert(action));
```

Reply:
My bad. I'm pretty sure I tried to compile it. Probably happened after.